acceptance-tests/src/main/java/io/blueocean/ath/AthModule.java

```diff
@@ -29,12 +32,16 @@
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.Properties;
+import java.util.logging.Level;
 
 public class AthModule extends AbstractModule {
     @Override
     protected void configure() {
 
-        DesiredCapabilities capability = DesiredCapabilities.firefox();
+        DesiredCapabilities capability = DesiredCapabilities.chrome();
```

Comment:
We should really pass this in from tests so we can run on multiple browsers at some point

Reply:
We also need to take a look at selenium-grid as it's the "selenium way" of firing off multiple test runs in different browsers.